### ISSUE TITLE
RE-J10 "Wolfhound/Schnauzer" Liquid Fuel Engine AJ10 - Gimbal not present in .mu

### ISSUE THREAD

**Issue by Poodmund** [PorktoberRevolution/ReStocked#737]
The Wolfhound, according to the part info, is supposed to have a 3 degree gimbal but it seems the gimbal is not set up in the .mu

No gimbal animation is present and the trust transform doesn't seem to change position when the engine is staged and you try to reorient the craft.

(Gimbal issue is an 'Art Bug', right?

**Comment by Poodmund** [comment]
Ah found it. The gimbal transform was incorrectly named in the config.

```  @MODULE[ModuleGimbal]
  {
    @gimbalTransformName = B_GimbalWH
  }
```
should be
```  @MODULE[ModuleGimbal]
  {
    @gimbalTransformName = B_Gimbal_WH
  }
```

**Comment by Poodmund** [comment]
https://github.com/PorktoberRevolution/ReStocked/commit/cc87ee1c2b12008797b01006018435ab240a00d2 gets the gimbal animation working but it seems the thrustTransform isn't parented under the mnoving gimbal game object...(?)

**Comment by ChrisAdderley** [comment]
That's not the right object ;)

**Comment by ChrisAdderley** [comment]
Fixed in a5cefc1b3d5d35f6ed3ab49d6534181ca1f9a932

**Comment by Poodmund** [comment]
Gimbal is present in all variants. Fine in 1.875m, 2.5m and Compact variants but the Classic variant's gimbal struts don't attach to any structure as the support frame is not present on the Classic variant. See image below:

![Untitled-1](https://user-images.githubusercontent.com/10106634/68152583-d993b480-ff3b-11e9-987d-65c87b72e86b.png)

**Comment by ChrisAdderley** [comment]
Now fixed in 2e33fbc806ec38cc3eb8ed13cad9c70a40d570b6

**Comment by Poodmund** [comment]
Resolved with https://github.com/PorktoberRevolution/ReStocked/commit/2e33fbc806ec38cc3eb8ed13cad9c70a40d570b6
